newsroom/decorator.py

```diff
@@ -33,10 +32,7 @@ def decorated_function(*args, **kwargs):
 def account_manager_only(f):
     @wraps(f)
     def decorated_function(*args, **kwargs):
-        if session.get("user_type") not in [
-            "administrator",
-            "account_management",
-        ] or not is_valid_login(session.get("user")):
+        if not is_current_user_admin() and not is_current_user_account_mgr() or not is_valid_session():
```

Comment:
It might be good to group these first two conditions in parentheses, so it makes it easier to read the order of operations.